docs/source/routing/performance/query-batching.mdx

```diff
@@ -1,16 +1,24 @@
 ---
 title: Query Batching
 subtitle: Receive query batches with the GraphOS Router
-description: Handle multiple GraphQL requests with GraphOS Router's query batching capabilities. Aggregate operations into single HTTP requests, reducing overhead.
+description: Handle multiple GraphQL requests with GraphOS Router's query batching capabilities. Aggregate operations into single HTTP requests, preserving data consistency and reducing performance overhead.
 ---
 
 <EnterpriseFeature />
 
-Learn about query batching and how to configure the GraphOS Router to receive query batches.
+Learn how query batching can help preserve data consistency of responses, and learn how to configure the GraphOS Router to receive query batches.
 
-## About query batching
+## About query batching 
 
-Modern applications often require several requests to render a single page. This is usually the result of a component-based architecture where individual micro-frontends (MFE) make requests separately to fetch data relevant to them. Not only does this cause a performance overhead—different components may be requesting the same data—it can also cause a consistency issue. To combat this, MFE-based UIs batch multiple client operations, issued close together, into a single HTTP request. This is supported in Apollo Client and Apollo Server.
+Modern applications often require several requests to render a single page. This is usually the result of a component-based architecture where individual micro-frontends (MFE) make requests separately to fetch data relevant to them. 
```

Comment:
We should probably add an extra line here saying this is not a performance optimisation, but one of the solutions to data consistency.

Reply:
```suggestion
Modern applications often require several requests to render a single page. This is usually the result of a component-based architecture where individual micro-frontends (MFE) make requests separately to fetch data relevant to them. 

Query batching makes response data consistent for MFEs making multiple requests per page. It primarily preserves data consistency rather than improve performance.
```